### ISSUE TITLE
UI Element Preferences Reset on Save Launch

### ISSUE THREAD

**Issue by DingoDjango** [RimWorldCCLTeam/CommunityCoreLibrary#171]
Filing a bug from A14 that I experienced with CCL and CCL mods (ex. FluffyTabs).

When loading a saved game UI elements would reset their state. This was the case even when loading from an on-going session, so I suppose this has something to do with the save loading mechanics.

For example - "auto-expand home zone" would reset to ON (default) even if I had set it to off multiple times in the past. Also, work priorities would reset to automatic. Work priorities would be saved so that turning them back to manual mode retained the values in the save file.

Having automatically downloaded A15 with Steam I have deleted my saved games and mods, so unfortunately I cannot provide a reliable reproduction of this bug with a saved file / mod list...

**Comment by BlueWinds** [comment]
Just wanted to confirm this issue - I've been seeing the same thing since I updated CCL from 14.2.1 -> 14.3.1. I did update a couple of other mods at the same time, so can't 100% guarantee that it's CCL's fault, but seems likely.

Repro is simple:
- Start new game.
- Change to manual work priorities, adjust firefighting to P1.
- Save.
- Exit.
- Load save.
- Work priorities are back to automatic.
- Change to manual work priorities - firefighting is still P1.

**Comment by ForsakenShell** [comment]
Yes, this is 99% likely CCLs fault with the ToggleSettingDefs (which controls those) being added.  I must have mucked up a little bit of code.  I'll make sure it's fixed for A15.

**Comment by mordang7** [comment]
Same here, after CCL 14.3.1 and I think 14.3 not only manual priorities but other mod settings will not save their settings, even the minimap won't save preferences.
